open-metadata-implementation/user-interfaces/presentation-server/cra-client/public/index.html

```diff
@@ -5,7 +5,7 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="shortcut icon" href="%PUBLIC_URL%/egeria_logo.png">
```

Comment:
Can we reuse one of the existing egeria logos and keep the files in the same place, so that our branding is kept together?

If requried the extra file can be brought into the build tree by a maven step once we integrate the build (getting resources)

Reply:
@planetf1 How important would you say it is to keep this file in the same directory as all the others? It is a bit of pain to work with references in the `index.html` in React (but doable).

I do think we should use the new logo. I think the color stands out the best on the browser tab compared to others, which are hard to make out.

Reply:
I don't like the idea of duplication. the project as a whole should be coherent. The standard way of doing this would be - if needed - to copy in the required resources using maven (or gradle), probably better than relative links which can be problematic.

As such I suggest we defer to a seperate issue to resolve - leave as is for now.

Reply:
see #3576